### DISCUSSION TITLE
Cant click on keep changes when changing the second to first monitor

### DISCUSSION THREAD

**Discussion by ngohungphuc** [utmapp/UTM#2634] [UTM Help (macOS)]
**Describe the issue**
A clear and concise description of what the issue is. **BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Configuration**
* UTM Version: 2.1.1
* OS Version: 
Apple Silicon

When I followed the step 
```
Scroll down to “Multiple displays” and select “Show only on 1”. The mouse might be stuck in the second display so you will have to use the keyboard to confirm and then restart.
```

I cant click on anything even trying to type something the screen is stuck there how can I click on keep changes ?

<img width="518" alt="Schermafbeelding 2021-06-17 om 23 19 57" src="https://user-images.githubusercontent.com/13591213/122436509-130b1d80-cfc3-11eb-9337-33a19c7227ae.png">

**Comment by luserppc** [comment]
I'm also having this issue.
Hardware is an M1 MacBook Air, running 11.4
On the latest UTM & the windows ver is 21354.  I wasn't having much success w/ the VHDX extension so I followed the guide, installed QEMU via Homebrew,  and converted my image to a qcow2 one.  

Just to add to the above description of the issue; even when using a keyboard (tab & arrows) to select 'keep changes'  the VM freezes, and will not set the monitor to Display #1.

Update 6/18: I remounted the spice tools iso, and re-installed spice-guest-tools0.164
This seems to have immediately  fixed my issue.

**Reply by neekey** [reply]
hi @luserppc can you elaborate how and when do you do that to solve this issue?

**Comment by ngohungphuc** [comment]
@luserppc now my mouse turn into a circle and I cant click 

![Schermafbeelding 2021-06-20 om 09 53 07](https://user-images.githubusercontent.com/13591213/122660692-5fb24c80-d1ad-11eb-8cb1-6c1f026d2ed6.png)
